src/config/resharding.rst

```diff
@@ -77,3 +87,22 @@ Resharding Configuration
 
             [reshard]
             source_close_timeout_sec = 600
+
+    .. config:option:: require_node_param
+
+        Require users to specify a ``node`` parameter when creating resharding
+        jobs. This can be used as a safety check to avoid inadvertently
+        starting too many resharding jobs by accident::
+
+
+            [reshard]
+            require_node_param = false
+
+    .. config:option:: require_range_param
+
+        Require users to specify a ``node`` parameter when creating resharding
```

Comment:
This looks like a duplicate of the last paragraph. Wrong description?

Reply:
It was a copy-paste error. It was supposed to be range instance of node. I'll update it.